.github/workflows/deploy_production.yml

```diff
@@ -40,20 +40,23 @@ jobs:
             az storage blob upload \
               --account-name zooniversestatic \
               --content-cache-control 'public, max-age=60' \
+              --overwrite
```

Comment:
going back to this..dont think this is the underlying issue but how important is the '\'?   @zwolf

Reply:
extremely 🤦